Python/initconfig.c

```diff
@@ -130,7 +130,10 @@ The following implementation-specific options are available:\n\
 \n\
 -X int_max_str_digits=number: limit the size of int<->str conversions.\n\
     This helps avoid denial of service attacks when parsing untrusted data.\n\
-    The default is sys.int_info.default_max_str_digits.  0 disables."
+    The default is sys.int_info.default_max_str_digits.  0 disables.\n\
+\n\
+-X cpu_count=number: limit cpu count of os.cpu_count.\n\
```

Comment:
If you prefer to write "number" instead of "n", please update the whole PR for that.

```suggestion
-X cpu_count=n: limit CPU count of os.cpu_count().\n\
```

Why is it a "limit"? Your unit test shows that it can be BIGGER than the real number of logical CPUs of the system!

Maybe just say "override"?